python/ray/workflow/workflow_storage.py

```diff
@@ -57,8 +57,7 @@
 
 # TODO: Get rid of this and use asyncio.run instead once we don't support py36
```

Comment:
This comment should be removed too :)

Reply:
(just randomly saw this, you probably planned to do that anyways)

Reply:
Hi @pcmoritz  this is just a PR to show that simply using `asyncio.run` won't work. I think I'll close this.